UC-Berkeley-Extension/global
# Quick tabs

## Asana Ticket(s)
- See ticket https://app.asana.com/0/1174970560648905/1196473750676361

## Description
- Adds tab functionality using Layout Builder Tabs
- Adds patch to the module to remove errors
- Adds Layout Builder Modal module for better UI
- Minor styling addition to LB form

## Testing instructions
- `ddev start`, `ddev ssh`, `composer install`, `drush cim`, `drush cr`
- Create a layout page, then edit the layout.
- Add a section, find the tabs option.
<img width="300" alt="Screen Shot 2020-11-02 at 2 38 02 PM" src="https://user-images.githubusercontent.com/10488517/97916778-16de7e00-1d19-11eb-9cdb-91db1e0b5305.png">

- In the section, add blocks in order you'd like the tabs to show. Save the layout, and confirm that the blocks are showing as tabs.
<img width="800" alt="Screen Shot 2020-11-02 at 2 22 41 PM" src="https://user-images.githubusercontent.com/10488517/97916875-3aa1c400-1d19-11eb-94da-e72fad81cc38.png">
